web_external/TreeView/TrackPane/index.jsx

```diff
@@ -15,19 +15,36 @@ class TrackPane extends BasePane {
     }
 
     getItemId(item) {
-        return item.id1;
+        return item;
```

Comment:
It's a little bit weird for Track, for activity it is the id but the track is the trackId itself. It's a function from the base class, I think it can be left as it is for now.